dataland-frontend/tests/component/components/resources/frameworkDataSearch/lksg/LksgPanel.cy.ts

```diff
@@ -0,0 +1,21 @@
+import LksgPanel from "@/components/resources/frameworkDataSearch/lksg/LksgPanel.vue";
+import { LksgData } from "@clients/backend";
+import { mount } from "cypress/vue";
+
+describe("Component test for LksgPanel", () => {
+  it("Should display the total revenue kpi in the correct format", () => {
+    const pseudoLksgData = { social: { general: { dataDate: "2023-01-01", totalRevenue: 1234567.89 } } };
```

Comment:
Here the testdata is defined directly in the test, right?

Reply:
Yes, in this minimal way.

Reply:
Ok